scripts/local/integration-tests/erc20_bridge_multihop.sh

```diff
@@ -39,19 +43,22 @@ native_erc20_deploy_result=$(forge create --private-key $user_private_key src/Mo
 native_erc20_contract_address=$(parseContractAddress "$native_erc20_deploy_result")
 echo "Test ERC20 contract deployed to $native_erc20_contract_address on Subnet A"
 
-# Deploy the ERC20 bridge contract to both chains.
-bridge_a_deploy_result=$(forge create --private-key $user_private_key --constructor-args $teleporter_contract_address \
-    --rpc-url $subnet_a_url src/CrossChainApplications/ERC20Bridge/ERC20Bridge.sol:ERC20Bridge)
+# Deploy the ERC20 bridge contract to all chains.
+bridge_a_deploy_result=$(forge create --private-key $user_private_key \
```

Comment:
Was there a need to move `--constructor-args` to be the last argument here with the last foundry version, or did you just move it to be consistent in our usage?

Reply:
Was not due to last version, but on their page they say "The --constructor-args flag must be positioned last in the command, since it takes multiple values.", even though it seemed to work previously without being last position. https://book.getfoundry.sh/reference/forge/forge-create